l5kit/l5kit/sampling/agent_sampling.py

```diff
@@ -100,9 +98,9 @@ def generate_agent_sample(
     cur_agents = history_agents[0]
 
     if selected_track_id is None:
-        agent_centroid = cur_frame["ego_translation"][:2]
-        agent_yaw = rotation33_as_yaw(cur_frame["ego_rotation"])
-        agent_extent = np.asarray((EGO_EXTENT_LENGTH, EGO_EXTENT_WIDTH, EGO_EXTENT_HEIGHT))
+        agent_centroid_m = cur_frame["ego_translation"][:2]
+        agent_yaw_rad = rotation33_as_yaw(cur_frame["ego_rotation"])
```

Comment:
Seen it a couple times now until here: do we sometimes use degrees or why is _rad necessary?

Reply:
it's for clarity and easy documentation :)